### ISSUE TITLE
Pictures are not displayed in pdf generated in Proposals, Orders and invoices

### ISSUE THREAD

**Issue by edupulpillo** [Dolibarr/dolibarr#17919]
Even configuring properly the options in setup/other setup to display pictures of the products in the PDF files generated, the pictures are missing
This is the summary of the bugs, where the pictures are not included in PDF generated files: 

1. MAIN_GENERATE_INVOICES_WITH_PICTURE=1	KO Do not works with customer invoices and KO do not works with vendor invoices.
2. MAIN_GENERATE_ORDERS_WITH_PICTURE=1	KO Do not  works with customer orders
3. MAIN_GENERATE_PROPOSALS_WITH_PICTURE=1	OK with customer proposals
4. MAIN_GENERATE_SUPPLIER_INVOICES_WITH_PICTURE=1 KO, do not works with vendor invoices
5. MAIN_GENERATE_SUPPLIER_ORDER_WITH_PICTURE=1	KO, do not works with vendor purchase orders
6. MAIN_GENERATE_SUPPLIER_PROPOSAL_WITH_PICTURE=1	KO do not works with vendor proposals/requests

**Comment by ksar-ksar** [comment]
Hello,

Not sure that Git is the best place to discuss, better to go on forums.

What is your dolibarr version ?
You PHP version ?
The dolibarr log ?
etc...

**Comment by KlaasOl** [comment]
This is an very old issue. I had the same in version 9,10,11. Then is was solved. Thanks to this fix: [https://github.com/Dolibarr/dolibarr/pull/13545/files](url)
Although problem comes again with version 12 and 13.
I did the same as edupulpillo.

Now I am using:

Dolibarr:
Huidige versie van Dolibarr (Programs) | 13.0.2 &nbsp; Zie ChangeLog bestand (alleen in het Engels) &nbsp; &nbsp; - &nbsp; &nbsp; Laatste stabiele versie : Controleren
Laatste versie upgrade (Database) | 13.0.2
Initiële installatie versie | 9.0.2

PHP:
Versie | 7.0.33-50+ubuntu16
GET and POST support | Deze PHP installatie ondersteunt POST en GET.
Sessions support | Deze PHP installatie ondersteund sessies.
UTF-8 support | Deze PHP ondersteunt UTF8-functies.

Anyone?
